apps/system/utils/kdbg_date.c

```diff
@@ -127,6 +127,9 @@ static inline int date_settime(int argc, char **args)
 	FAR char *saveptr;
 	long result;
 	int ret = OK;
+#ifdef CONFIG_LIBC_LOCALTIME
+	tm.tm_isdst = 0;
```

Comment:
Is it not required to parse the parameters of date_settime and set all the members associated with CONFIG_LIBC_LOCALTIME) || CONFIG_TIME_EXTENDED accordingly?

-----
struct tm {                                                                      
        int tm_sec;                                     /* second (0-61, allows for leap seconds) */
        int tm_min;                                     /* minute (0-59) */         
        int tm_hour;                            /* hour (0-23) */                
        int tm_mday;                            /* day of the month (1-31) */       
        int tm_mon;                                     /* month (0-11) */          
        int tm_year;                            /* years since 1900 */           
#if defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED)              
        int tm_wday;                            /* day of the week (0-6) */         
        int tm_yday;                            /* day of the year (0-365) */       
        int tm_isdst;                           /* non-0 if daylight savings time is in effect */
#endif                                                                           
};